### DISCUSSION TITLE
Announcing `bevy-persistent-windows`: A Bevy plugin to easily create and manage windows that remember where they were.

### DISCUSSION THREAD

**Discussion by umut-sahin** [bevyengine/bevy#9423] [Show and tell]
Hello Bevy Community!

I've created `bevy-persistent-windows` to make windows exactly restore to where they were in the previous run. To me, this is very useful, especially during development with frequent restarts to the game. Multiple monitors are supported!

- Crate: https://crates.io/crates/bevy-persistent-windows
- Documentation: https://docs.rs/bevy-persistent-windows
- Repository: https://github.com/umut-sahin/bevy-persistent-windows
- Examples: https://github.com/umut-sahin/bevy-persistent-windows/tree/main/examples

Feel free to give feedback, suggest ideas, and contribute!

**Comment by killercup** [comment]
You might want to add it to https://github.com/bevyengine/bevy-assets too :)

**Reply by umut-sahin** [reply]
Thanks for the suggestion! PR is up https://github.com/bevyengine/bevy-assets/pull/350 :partying_face:
